src/app/clusters/operational-state-server/operational-state-server.cpp

```diff
@@ -106,12 +106,12 @@ void OperationalStateServer::HandlePauseState(HandlerContext & ctx, const Comman
     Commands::OperationalCommandResponse::Type response;
     Delegate * delegate = OperationalState::GetOperationalStateDelegate(mEndpointId, mClusterId);
     GenericOperationalError err(to_underlying(ErrorStateEnum::kNoError));
-    GenericOperationalState opState;
+    uint8_t opState;
```

Comment:
Should really be declared down where it's first used, so:
```
   uint8_t opState =  delegate->GetCurrentOperationalState();
```
down after we check the delegate is not null.  Similar in other places.